islandora_transforms/slurp_XML_converted_JSON_to_solr.xslt

```diff
@@ -8,8 +8,8 @@
     <xsl:param name="prefix" select="concat('json_', ../@ID, '_')"/>
     <xsl:param name="suffix">ms</xsl:param>
     <xsl:param name="root_node">json</xsl:param>
-
-    <xsl:apply-templates mode="index_converted_json" select="$content/*[local-name() = $root_node]">
+    
+    <xsl:apply-templates mode="index_converted_json" select="$content/$root_node">
```

Comment:
This seems to be a regression https://github.com/discoverygarden/islandora_transforms/commit/cff21c230ba279bcadeb79ce74492c0c632d4480